After signaling SIGINT during npm run dev, console throws errors

# Bug report

While trying for the first time `nextjs` got this error during development. Is this normal?

## Describe the bug

After signaling `SIGINT` with `control+c` in the console during `npm run dev` the console throws some errors.

## To Reproduce

Steps to reproduce the behavior:

1. Initialize npm
`npm init -y`
2. Install the core NextJS dependencies:
`npm install --save next react react-dom`
3. Add scripts to the `package.json` file:
```json
"scripts": {
    "dev": "next",
    "build": "next build",
    "start": "next start"
}
```
4. Create `pages/index.js` file:
```js
import React from "react";

export default () => {
  return (
    <div>
      <p>Hello world</p>
    </div>
  );
}
```
5. Start `npm run dev`:
```bash
 DONE  Compiled successfully in 1801ms

> Ready on http://localhost:3000
```
6. Press `control+c` in the console (this is the buggy behavior):
```bash
^Cgot signal SIGINT, exiting
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! test@1.0.0 dev: `next`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the test@1.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/Felipe/.npm/_logs/2018-08-07T17_37_32_226Z-debug.log
```

## Expected behavior
I think there shouldn't be any errors while gracefully exiting the app during development.

## System information
 - OS: macOS
```json
{
  "name": "test",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "dev": "next",
    "build": "next build",
    "start": "next start"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "next": "^6.1.1",
    "react": "^16.4.2",
    "react-dom": "^16.4.2"
  }
}
```

## Additional context

`debug.log` output:

```log
8 verbose lifecycle test@1.0.0~dev: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/Fe    lipe/Local_Projects/Aprender/nextjs/test/node_modules/.bin:/Users/Felipe/anaconda3/bin:/usr/local/sbin:/usr/local/opt/fzf/b    in:/Users/Felipe/google-cloud-sdk/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin
9 verbose lifecycle test@1.0.0~dev: CWD: /Users/Felipe/Local_Projects/Aprender/nextjs/test
10 silly lifecycle test@1.0.0~dev: Args: [ '-c', 'next' ]
11 silly lifecycle test@1.0.0~dev: Returned: code: 1  signal: null
12 info lifecycle test@1.0.0~dev: Failed to exec dev script
13 verbose stack Error: test@1.0.0 dev: `next`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:304:1    6)
13 verbose stack     at EventEmitter.emit (events.js:182:13)
13 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:5    5:14)
13 verbose stack     at ChildProcess.emit (events.js:182:13)
13 verbose stack     at maybeClose (internal/child_process.js:961:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:5)
14 verbose pkgid test@1.0.0
15 verbose cwd /Users/Felipe/Local_Projects/Aprender/nextjs/test
16 verbose Darwin 17.7.0
17 verbose argv "/usr/local/Cellar/node/10.7.0/bin/node" "/usr/local/bin/npm" "run" "dev"
18 verbose node v10.7.0
19 verbose npm  v6.3.0
20 error code ELIFECYCLE
21 error errno 1
22 error test@1.0.0 dev: `next`
22 error Exit status 1
23 error Failed at the test@1.0.0 dev script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

Looked for any similar error but couldn't find any. If this is a duplicate please excuse me.

--- a/bin/next
+++ b/bin/next
@@ -75,7 +75,7 @@ const startProcess = () => {
         process.exit(137)
       }
       console.log(`got signal ${signal}, exiting`)
-      process.exit(1)
+      process.exit(signal === 'SIGINT' ? 0 : 1)
     }
     process.exit(0)
   })

